### ISSUE TITLE
Question: Can this app be via console only?

### ISSUE THREAD

**Issue by AaronJWhite** [anetczuk/LinakDeskApp#10]
Can this app be via console only? If so, could you add the commands to the documentation?

**Comment by anetczuk** [comment]
Hello, what do You mean by "console only"? Something like execute single command through cmd and exit?

**Comment by AaronJWhite** [comment]
Yes, I would be interested in running this from a headless raspberry pi by logging in with ssh and running a few commands like up, down, set position 32.2, etc.

**Comment by anetczuk** [comment]
Check #1 -- someone hacked simple cmd script allowing to change position.
Moreover there is `WebServer` tab allowing to start web server and control positioning through http.

**Comment by AaronJWhite** [comment]
Is there documentation for how the web server is supposed to work? I enabled it, but it didn't seem to be loading and I don't know what the api/commands would be.

**Comment by anetczuk** [comment]
First of all go to `WebServer` tab and activate the checkbox. Then open http://localhost:8080 and read description about available commands (description added few minutes ago).
For now there is only possibility to move desk to predefined favorite position.
